Workbooks/CosmosDb/Overview/Overview.workbook

```diff
@@ -445,4 +445,4 @@
   ],
   "styleSettings": {},
   "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
-}
+}
```

Comment:
weird. how is this a line change? Extra spacing maybe?

Reply:
I think, there was a extra space after the '}', that got removed now.